docs/changelogs/CHANGELOG-2.21.md

```diff
@@ -0,0 +1,466 @@
+
+- Add controllers for calculating resource quota global and local seed usage ([#10160](https://github.com/kubermatic/kubermatic/pull/10160))
+- Clusters will now have resource usage in in their `status` ([#10070](https://github.com/kubermatic/kubermatic/pull/10070))
+- Add a Machine validating webhook which checks the Machine resource requests (CPU, Memory, Storage) against its project's resource quota (if set) ([#9650](https://github.com/kubermatic/kubermatic/pull/9650))
+- Introduce new API endpoints for CRUD operations on KKP Resource Quotas ([#10079](https://github.com/kubermatic/kubermatic/pull/10079))
+- Add admin setting page for adding resource quota for projects ([#4641](https://github.com/kubermatic/dashboard/pull/4641))
+- Add quota widget for projects in the dashboard ([#4731](https://github.com/kubermatic/dashboard/pull/4731))
+- Add support for managing project resource quotas in the dashboard ([#4680](https://github.com/kubermatic/dashboard/pull/4680), [#4690](https://github.com/kubermatic/dashboard/pull/4690))
+
+#### Applications
+​
+- Add application installation and management from Git/Helm sources ([#9977](https://github.com/kubermatic/kubermatic/issues/9977), [#10363](https://github.com/kubermatic/kubermatic/issues/10363))
+- Add API endpoints for ApplicationInstallations/ApplicationDefinitions ([#10286](https://github.com/kubermatic/kubermatic/issues/10286), [#10341](https://github.com/kubermatic/kubermatic/issues/10341))
+- Add authentication with RegistryConfigFile in HelmCredentials ([#10564](https://github.com/kubermatic/kubermatic/issues/10564), [#10570](https://github.com/kubermatic/kubermatic/issues/10570))
+
+#### Operating System Manager
+
+- Operating System Manager is enabled by default and it's responsible for creating and managing the required configurations for worker nodes ([#10415](https://github.com/kubermatic/kubermatic/pull/10415))
+- Containerd container runtime mirror registries support ([#10134](https://github.com/kubermatic/kubermatic/pull/10134))
+- OSM Deployment Docker image repository and tag can be overwritten using the `KubermaticConfiguration` ([#10123](https://github.com/kubermatic/kubermatic/pull/10123))
+
+#### Group Project Bindings (EE)
+
+- Add `GroupProjectBinding` CRD ([#10158](https://github.com/kubermatic/kubermatic/pull/10158))
+- Add REST API for interacting with GroupProjectBinding resources ([#10303](https://github.com/kubermatic/kubermatic/pull/10303), [#4712](https://github.com/kubermatic/dashboard/pull/4712))
+- Add support for GroupProjectBindings in Alertmanager authorization server ([#10574](https://github.com/kubermatic/kubermatic/pull/10574))
+
+#### Encryption At Rest
+
+- Add experimental support for encryption-at-rest with secretbox for static key encryption ([#9654](https://github.com/kubermatic/kubermatic/pull/9654))
+
+#### External Clusters
+
+- Add support for KubeOne External Cluster ([#9179](https://github.com/kubermatic/kubermatic/pull/9179))
```

Comment:
@imharshita IIRC KubeOne External Clusters are not going to be released. Should this one (and the following line) be removed?